### ISSUE TITLE
Can we improve the Introduction / Need decription?

### ISSUE THREAD

**Issue by zimeon** [OCFL/spec#650]
The idea and most of the [Need](https://ocfl.io/1.1/spec/#need) section is really good. I think there are opportunities to improve two of the five requirements descriptions:
* Versioning - I'm not sure how accessible the term "forward delta" is, it would not take much to be explicit and say we store only new and changed files. Also, we don't mention that this is an efficient strategy wrt storage use.
* Storage diversity - Should we be bold and say something about our hope that the simple approach that works with the range of current options hopefully sets us up with a good chance of working well on future storage technologies?

**Comment by awoods** [comment]
I agree that spelling out the versioning approach of storing new and changed files with a parenthetical mention of "forward delta" would be clarifying. 

Also, the "storage diversity" block could be bolstered.

**Comment by neilsjefferies** [comment]
With V2 we will have improved support for WORM media and object stores...
